content/nginx-one/connect-instances/connect-using-squid.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Prepare - Set up Squid as a proxy
+toc: true
+weight: 250
+nd-docs: DOCS-000
+---
+
+NGINX Agent can be configured to connect to NGINX One using a [Squid proxy](https://www.squid-cache.org/). This is useful in environments where direct internet access is restricted or monitored.
+
+## Before you start
+
+Ensure you have the following:
+
+- [Squid proxy server set up and running](https://wiki.squid-cache.org/SquidFaq/InstallingSquid)
+- [NGINX Agent is installed]({{< ref "nginx-one/agent/install-upgrade/" >}})
+- Access to the NGINX One console
+
+
+## Configure Squid
+
+Follow the steps below to configure Squid with basic authentication.
+
+1. Open the Squid configuration file with your favorite text editor (you might need superuser privileges):
+
+   ```sh
+   vi /etc/conf/squid.conf
```

Comment:
After installing Squid on Ubuntu:

```suggestion
   vi /etc/squid/squid.conf
```

If you trust the Squid wiki and build from source, the squid.conf file is at /usr/local/squid/etc/squid.conf